Libraries/Opc.Ua.Security.Certificates/System.Security.Cryptography.X509Certificates/X509SignatureGenerator.cs

```diff
@@ -0,0 +1,64 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if !NETSTANDARD2_1
+
+// This source code is intentionally copied from the .NET core runtime to close
+// a gap in the .NET 4.6 and the .NET Core 2.x runtime implementations.
+// original code is located here:
+// https://github.com/dotnet/runtime/blob/master/src/libraries/System.Security.Cryptography.X509Certificates/src/System/Security/Cryptography/X509Certificates/X509SignatureGenerator.cs
+#pragma warning disable CS1591 // Suppress missing XML comments to preserve original code
+
+using System;
+using System.Security.Cryptography;
+using System.Security.Cryptography.X509Certificates;
+
+namespace Opc.Ua.Security.Certificates
+{
+    public abstract class X509SignatureGenerator
+    {
+        private PublicKey _publicKey;
+
+        public PublicKey PublicKey
+        {
+            get
+            {
+                if (_publicKey == null)
+                {
+                    _publicKey = BuildPublicKey();
+                }
+
+                return _publicKey;
+            }
+        }
+
+        public abstract byte[] GetSignatureAlgorithmIdentifier(HashAlgorithmName hashAlgorithm);
+        public abstract byte[] SignData(byte[] data, HashAlgorithmName hashAlgorithm);
+        protected abstract PublicKey BuildPublicKey();
+#if NOT_SUPPORTED
```

Comment:
`NOT_SUPPORTED` is not defined in PR and only used in this class

Reply:
on purpose ECC is not supported for net462 amd standard2.0